process.remove_subprocess warning

cjcardinale opened the pull request:
Warning message added in a place of a KeyError if a subprocess, not in the subprocess dictionary, is removed. Verbose option was added, with a default: Verbose = True.  
The actual message is a suggestion; I wanted to keep it short, but it may be useful to explicitly tell the user that the subprocess is not in the subprocess dictionary instead of the current message:

> WARNING: Subprocess does not exist

These changes do not seem to conflict with any of the documentation on remove_subprocess, but additional writing about the verbose option could be introduced.

This is in reference to issue #21 .

brian-rose commented on climlab/process/process.py:
```diff
@@ -297,7 +297,11 @@ def remove_subprocess(self, name):
+            self.subprocess.pop(name)
+        except KeyError:
+            if verbose: 
+                print 'WARNING: Subprocess does not exist'
```
Let's make the warning more specific by telling the user the name that was not found, e.g.
```
print 'WARNING: ' + name + ' not found in subprocess dictionary.'
```

brian-rose replied:
Or even better because it uses proper Python string formatting:
```
print 'WARNING: {} not found in subprocess dictionary.'.format(name)
```

brian-rose commented:
We should try hard to keep the docstrings up to date. Here, since you've added the `verbose` argument, that should also be added to docstring.

brian-rose commented:
By the way, to make changes to this pull request before it gets accepted and merged, just push your new code changes to this same branch on github and everything should show up in the pull request without any further action. I think...

cjcardinale commented:
Yes, the code changes showed up in the pull request. I adjusted the warning message and added verbose to the docstring (the writing is consistent with other areas where verbose was included).

brian-rose commented:
Great. Done!
